### PR TITLE
Update cause modeling document

### PR DESCRIPTION
**Pull request by yongqx**
Make smaller PR for cause model section

### CODE REVIEW

File: docs/source/models/cause.rst [hunk 1]
```diff
@@ -4,66 +4,119 @@
...
 
-   Overview of modeling GBD causes.
-
-.. contents:
-
-What is a cause?
-----------------
+   Vivarium has the infrastructure to model causes at any level in the GBD hierarchy.
+   However, it is recommended to model GBD causes at the most detailed level to avoid 
+   problems with missingness in the GBD cause hierarchy.
```
Comment (aflaxman) [inline docs/source/models/cause.rst]: I recommend active voice, and saying why more specifically:
> If you model a cause at the most-detailed level, there is a better chance of all necessary data being available in GBD.  For example, cause "CKD due to Diabetes" has a relative risk for SBP, but less-detailed cause "CKD" does not have a relative risk for SBP (and we need to derive this RR from the PAF to use this less-detailed cause and this risk factor together in our hypertension treatment model).

File: docs/source/models/cause.rst [hunk 2]
```diff
@@ -4,66 +4,119 @@
...
 The structure of a cause model
 ------------------------------
 
+**Cause entities:** Searching the GBD cause_id associated with the cause. Find out
+affected outcomes by this cause (either mortality, disability, or both). Start thinking 
+about the restrictions such as certain ages or sexes applied to this cause.
+(useful package: `gbd_mapping <https://vivarium.readthedocs.io/projects/gbd-mapping/
+en/latest/gbd_mapping.html>`_)
+
+**Modeling Strategy:** To address proper methods that used to model causes we interested.
```
Comment (aflaxman) [inline docs/source/models/cause.rst]: "we interested" should be "we are interested in" or "in which we are interested".

File: docs/source/models/cause.rst [hunk 3]
```diff
@@ -4,66 +4,119 @@
...
+This decision will be driven by data availability and disease epidemiology. It's 
+recommended to document any cause with PAF of one relationship with modeled risks
```
Comment (aflaxman) [inline docs/source/models/cause.rst]: "It's recommended to document any cause ..." in active voice is "Document any cause ..."

File: docs/source/models/cause.rst [hunk 4]
```diff
@@ -4,66 +4,119 @@
...
 Common cause models
 -------------------
 
-.. todo::
-
-   Format as table with model type, description.
-   Fill in descriptions.
+**SI** - Short for susceptible-infected. This modeling strategy is appropriate for 
+causes where people never recover once they enter into a with condition state. 
+Example: Alzheimer’s disease and other dementias
```
Comment (aflaxman) [inline docs/source/models/cause.rst]: Let's have figures for each of these models.

File: docs/source/models/cause.rst (hunk 1 above)
Comment (yongqx) [reply docs/source/models/cause.rst]: Thank you Abie for more explicit description!

**Comment by collijk** [comment]
This PR has been superceded by the separate prs submitted by the rest of the research group.

**Pull request closed without merging**